src/collections/_documentation/performance/performance-tab.md

```diff
@@ -0,0 +1,46 @@
+---
+title: 'Performance Tab'
+sidebar_order: 1
+---
+
+[SCREENSHOT of Performance Homepage]
+
+The Performance Homepage offers Apdex and Throughput graphs as well as a list of the most problematic transactions so users can see a high-level report of their overall application health.
+
+## Apdex & Throughput Graphs
+The Apdex graph displays the ratio of response times in an application measured against a threshold. This information is displayed as an aggregate so you can get a sense of how long a customer has waited. For example, if the Apdex score dips significantly during a given time period, you may want to investigate that time period for a potential performance bottleneck.
+
+The Throughput graph indicates the number of transactions over a given time range. You can quickly visualize the traffic patterns in your application and see if a spike in throughput might be correlated with a dip in your Apdex score.
+
+## Transaction Table
+Underneath the graphs, you can view a list of your transactions with the capability to filter down on which transactions you want to view. In this table, you can sort on the provided columns to understand key characteristics of your transactions like [P95]({%- link _documentation/performance/performance-metrics.md -%}#p95-threshold) duration time, [Average Transaction Duration]({%- link _documentation/performance/performance-metrics.md -%}#average-transaction-duration) time, or [Throughput]({%- link _documentation/performance/performance-metrics.md -%}#throughput).
+
+## My Key Transactions
+"My Key Transactions" are transactions that you bookmark and can view in a separate dedicated tab. You may want to bookmark a Transaction as a "Key Transaction" if you monitor performance on a particular transaction or group of transactions. You can set a Transaction as a "Key Transaction" on the Transaction Summary page. The "All Transactions" view also displays these transactions.
+
+## Transaction Summary
+
+[ SCREENSHOT of Transaction Summary page ]
+
+Navigate to the Transaction Summary page by clicking on any transaction on the Performance Homepage. On the Transaction Summary page, users can find graphs, a list of slowest transactions, facet maps, and summary statistics about the transaction.
+
+## Transaction Summary Graphs
+Duration Breakdown: By graphing P50, P75, P95, P99, and P100 durations distinctly, users can utilize this display to see their transaction performance over time. Users can toggle the display of each segment by clicking on the legend (including Releases). Also, users can zoom in on specific slices to investigate spikes or possible performance regressions from a release.
```

Comment:
"you can utilize?" LMK if there's a reason we're adopting "users" here. Otherwise, we should make the change to "you" throughout.